help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,188 @@
+| | last_name | String | The contact's family name | Max Length: 50 characters |
+| | postal_code | String | The contact's ZIP code or other postal code. | |
+| | state_province_region | String | The contact's state, province, or region. | Max Length: 50 characters|
+| | line | String | The contact's Line account *(if any)* | |
+| | whatsapp | String | The contact's Whatsapp account *(if any)* | |
+| | facebook | String | The contact's Facebook account *(if any)* | |
+| | unique_name | String | | |
+| | phone_number | String | The contact’s phone number. | |
+
+* After completing the mappings, select **Next** to obtain a review screen indicating the audience and destination are connected.
+![](../../assets/catalog/email-marketing/sendgrid/22.png)
+
+* Select **Finish** to complete the setup.
+![](../../assets/catalog/email-marketing/sendgrid/23.jpg)
+
+## Validate data within Sendgrid {#validate}
+
+To validate that you have correctly set up the destination.
+
+1. Select **Destinations > Browse** to navigate to the list of destinations. 
+![](../../assets/catalog/email-marketing/sendgrid/25.jpg)
+
+1. Select the destination and ensure the status is **enabled**.
+![](../../assets/catalog/email-marketing/sendgrid/26.jpg)
+
+1. Switch to the **Activation data** tab, then select a **Segment Name**.
+![](../../assets/catalog/email-marketing/sendgrid/27.jpg)
+
+1. Monitor the segment summary and check the count of profiles corresponds to the count created within the dataset.
+![](../../assets/catalog/email-marketing/sendgrid/28.jpg)
+
+1. The [Sendgrid Marketing Lists API](https://api.sendgrid.com/v3/marketing/lists) is used to create unique contact lists within Sendgrid  by joining the value of the *list_name* attribute and the timestamp of the data export. Navigate to the Sendgrid site and check if the new contact list conforming to the name pattern is created.
+
+![](../../assets/catalog/email-marketing/sendgrid/29.jpg)
+![](../../assets/catalog/email-marketing/sendgrid/30.jpg)
+
+1. Select the newly created contact list and check if the new email record from the dataset you created is being populated within the new contact list.
+
+1. Additionally also check a couple of emails to validate if the field mapping is correct.
+![](../../assets/catalog/email-marketing/sendgrid/31.jpg)
+![](../../assets/catalog/email-marketing/sendgrid/32.jpg)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](https://experienceleague.adobe.com/docs/experience-platform/data-governance/home.html).
+
+## Additional resources {#additional-resources}
+
+This Sendgrid destination leverages the below API's
+  * [Sendgrid Marketing Lists API](https://api.sendgrid.com/v3/marketing/lists) 
```

Comment:
please edit the links in this and the next row to go out to proper Sendgrid documentation pages.